docs/developer-docs/latest/plugins/upload.md

```diff
@@ -166,11 +203,11 @@ const response = await fetch('http://localhost:1337/api/upload', {
 You have to send FormData in your request body.
 :::
 
-## Upload files related to an entry
+### Upload entry files
 
-To upload files that will be linked to a specific entry.
+Upload one or more files that will be linked to a specific entry.
 
-### Request parameters
+The following parameters are accepted:
```

Comment:
I think this would be more readable as a table, as we usually do for a list of parameters.